test/common_quantization.py

```diff
@@ -156,9 +268,9 @@ def __init__(self):
     def forward(self, x):
         return self.relu(self.fc2(self.relu(self.fc1(x))))
 
-class WrappedModel(torch.nn.Module):
+class SkipQuantModel(torch.nn.Module):
```

Comment:
If feel that if the name can be changed this significantly, then maybe it should have a comment explaining what it is.

Reply:
sure, it's just we changed API and previously only this model is using QuantWrapper, and now every model uses it, so need to pick something else to distinguish this model with others.